### PR TITLE
requestPresent throws an Error when VRLayers source is null (fix #72)

### LINKED ISSUE #72: VRDisplay.requestPresent should throw when VRLayer.source is null
I propose that we continue to allow VRLayer.source to be a null value, with the expectation that future extensions to VRLayer will perhaps define new kinds of layers that do not require a source value.

Perhaps VRDisplay.requestPresent should throw when it does not have enough information to create a valid VRLayer.  This would imply that until there are additional attributes to define new layer types, that VRDisplay.requestPresent would throw when VRLayer.source is null

### PR DESCRIPTION
**Pull request by dmarcos**
Trying to capture https://github.com/w3c/webvr/issues/72 opened by @kearwood

### CODE REVIEW

**Comment by kearwood** [comment]
LGTM, Thanks!

**Comment by cvan** [comment]
Thanks. Can you run `make`, rebase, and push again? (we'll add an auotmated GitHub Webhook in #18.)

File: index.bs [hunk 1]
```diff
@@ -140,7 +140,9 @@ interface VRDisplay : EventTarget {
   /**
    * Begin presenting to the VRDisplay. Must be called in response to a user gesture.
    * Repeat calls while already presenting will update the VRLayers being displayed.
+   * If the source of any of the layers is not present (null) requestPresent throws an Error
    */
+  [Throws]
   Promise&lt;void&gt; requestPresent(sequence&lt;VRLayer&gt; layers);
```
Comment (cvan) [inline index.bs]: Does `exitPresent` not throw as well?

File: index.bs [hunk 2]
```diff
@@ -140,7 +140,9 @@ interface VRDisplay : EventTarget {
   /**
    * Begin presenting to the VRDisplay. Must be called in response to a user gesture.
    * Repeat calls while already presenting will update the VRLayers being displayed.
+   * If the source of any of the layers is not present (null) requestPresent throws an Error
```
Comment (cvan) [inline index.bs]: can you add a comma after ``null)` and period at the end?

**Comment by toji** [comment]
Sorry, I skimmed over this previously, should have looked more closely. Why are we having this throw? Seems like the cleaner response is to reject the promise with an error message.

**Comment by dmarcos** [comment]
@kearwood Should we throw exceptions or reject the promise?

**Comment by kearwood** [comment]
I think we should reject the promise rather than throw exceptions

**Comment by dmarcos** [comment]
@kearwood I addressed the comments and updated the bikeshed output

**Comment by toji** [comment]
LGTM. Merging.

**Pull request merged**